### ISSUE TITLE
Generate load testing scripts with Gatling

### ISSUE THREAD

**Issue by jdubois** [jhipster/generator-jhipster#1111]
This one is for me, I already have the basic tests working, it's mostly a matter of templating them

**Comment by gmarziou** [comment]
I'm really curious about it.

**Comment by jimbobwyatt** [comment]
+1, automated performance tests protect this project and derived projects.

**Comment by atomfrede** [comment]
@jdubois Are you planning to integrate it into the maven build, so it can be executed on a nightly jenkins build?

**Comment by jdubois** [comment]
I have a lot of trouble to integrate it into Maven.

We might end up having a complete Gatling installation... I don't like it but it looks like it's how it's supposed to be used. I'm not a Gatling expert, so I don't want to go against the "normal" usage of the tool.

**Comment by cselti** [comment]
It's possible to execute it from Maven (including the recorder tool).
I have used the Gatling 2 with maven a couple of months before (only at basic level, I'm not a gatling expert too) and I followed the instruction on the gatling documentation site (http://gatling.io/docs/2.1.4/extensions/maven_plugin.html) + the sample project (https://github.com/gatling/gatling-maven-plugin-demo). 
Sadly, this part of the gatling isn't documented well at the moment - so I have checked sometimes the source code too (https://github.com/gatling/gatling-maven-plugin/tree/master/src/main/java/io/gatling/mojo) - but after I had configured successfully, I could use easily afterwards. The gatling is an awesome tool - like the JHipster :)  (I'm using the jhipster for 1 week only, but it rocks)
